### PR TITLE
22 ma sidebarbutton

### PR DESCRIPTION
**Pull request by Moon2024-light**
please review my coding for sidebar button styling thanks

### CODE REVIEW

File: styles/general.css [hunk 1]
```diff
@@ -196,3 +196,41 @@ footer {
...
+
+/* Sidebar Button Styling */
+.sidebar-button {
+    background-color: transparent;
+    border: none;
+    padding: var(--small-padding) var(--base-padding);
+    font-size: var(--base-font);
+    color: var(--textcolor-main);
+    cursor: pointer;
+    transition: all 0.3s ease; /* Smooth transition for hover effect */
```
Comment (TobiasKjernell) [inline styles/general.css]: can take away tranisition here since it will look goofy when you highlight them fast.

Comment (TobiasKjernell) [reply styles/general.css]: can also add border-radius: var(--border-radius)". We use that on other similiar buttons

File: styles/general.css [hunk 2]
```diff
@@ -196,3 +196,41 @@ footer {
 .medium-horizontal-padding {
  padding: 0 3rem;
 }
+
+
+
+/* Sidebar styling */
+.sidebar {
+    background-color: #F5F0D7; /* Main background for sidebar */
```
Comment (TobiasKjernell) [inline styles/general.css]: can take away this background color since we using texture

**Review by Moon2024-light — COMMENTED** [review]
please check

**Review by TobiasKjernell — APPROVED** [review]

**Review by idacarlin95 — APPROVED** [review]
looks good

**Pull request merged**